common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -77,22 +77,22 @@ private ArmEnvironment GetEnvironment(string endpoint)
         {
             if (string.IsNullOrEmpty(endpoint))
             {
-                return ArmEnvironment.AzureCloud;
+                return ArmEnvironment.AzurePublicCloud;
```

Comment:
All these cloud names are from the result of https://management.azure.com/metadata/endpoints?api-version=2019-11-01, I think we need follow this one but not from another sdk

Reply:
In an archboard chat today Ted said we should follow the Azure.Identity public API names here https://apiview.dev/Assemblies/Review/6a40db7f9eeb41dabfe5768f1e796c81#Azure.Identity.AzureAuthorityHosts

Reply:
Because of the Azure.Identity has GAed so we can't change it? If this is real then mgmt. sdk fixed at lot that not did right in released data plane sdk

Reply:
The url above is designed to be the source of the cloud endpoints settings for all Azure clients including SDKs and CLIs.

Reply:
@tg-msft can you confirm what you would like us to do here for naming?

Reply:
As offline discussed with @m-nash , let's merge it first